README.md

```diff
@@ -1,44 +1,92 @@
 # Prometheus SNMP Exporter
 
-This is an exporter that exposes information gathered from SNMP
-for use by the Prometheus monitoring system.
+This exporter is the recommended way to expose SNMP data in a format which
+Prometheus can ingest.
 
-There are two components. An exporter that does the actual scraping, and a
-[generator](generator/) (which depends on NetSNMP) that creates the
-configuration for use by the exporter.
+To simply get started, it's recommended to use the `if_mib` module with
+switches, access points, or routers.
+
+# Concepts
+
+While SNMP uses a hierarchical data structure and Prometheus uses an
+n-dimnensional matrix, the two systems map perfectly, and without the need
+to walk through data by hand. `snmp_exporter` maps the data for you.
+
+## Prometheus
+
+Prometheus uses so called labels attached to data to give identity to the
+data.
```

Comment:
I'm thinking it would be more useful to mention how SNMP maps to labels. Just mentioning the fact that labels exist doesn't help much.

How about this:

```suggestion
Prometheus is able to map SNMP index instances to labels. For example, the `ifEntry` specifies an INDEX of  `ifIndex`. This becomes the `ifIndex` label in Prometheus.

If an SNMP entry has multiple index values, each value is mapped to a separate Prometheus label.
```